test/integration/component/test_advancedsg_networks.py

```diff
@@ -169,6 +171,26 @@ def setUpClass(cls):
         # Get Zone, Domain and templates
         cls.domain = get_domain(cls.api_client)
         cls.zone = get_zone(cls.api_client, cls.testClient.getZoneForTests())
+        if cls.zone.securitygroupsenabled is False:
```

Comment:
component test won't run in trillian test

can you move to smoke ? if some tests are not really necessary, we can remove `"advanced"` from `@attr(tags`
if everything is fine, we can move back to component if it takes long time.

Reply:
As of now these tests take around 20 minutes. Let me see what all tests we can disable and move this file to smoke tests.

Reply:
@weizhouapache I ran the acceptance tests. And each tests seems to be taking around 3-5 minutes. All of these tests in total are taking around 2 hours. What all tests do you think we need to keep in smoke?

Class | Test Name | Time
--- | --- | ---
TestNetworksInAdvancedSG | test_03_createIsolatedNetwork | 211.864
TestNetworksInAdvancedSG | test_05_deployVM_SharedwithSG | 316.476
TestNetworksInAdvancedSG | test_06_SharedNwSGAccountSpecific | 218.468
TestNetworksInAdvancedSG | test_07_SharedNwSG_DomainWide_SubdomainAcccess | 306.024
TestNetworksInAdvancedSG | test_10_deleteSharedNwSGAccountSpecific_InUse | 314.851
TestNetworksInAdvancedSG | test_11_deleteSharedNwSG_DomainWide_InUse | 303.817
TestNetworksInAdvancedSG | test_12_deleteSharedNwSGAccountSpecific_NotInUse | 215.508
TestNetworksInAdvancedSG | test_13_deleteSharedNwSG_DomainWide_NotInUse | 212.807
TestNetworksInAdvancedSG | test_29_deleteSharedNwSG_ZoneWide_InUse | 305.857
TestNetworksInAdvancedSG | test_30_deleteSharedNwSG_ZoneWide_NotInUse | 211.688
TestNetworksInAdvancedSG | test**14_createSharedNwWithSG_withoutParams | 214.441
TestNetworksInAdvancedSG | test**15_createVPC | 264.207
TestNetworksInAdvancedSG_VmOperations | test_18_DeployVM_NoFreeIPs_1_account | 308.510
TestNetworksInAdvancedSG_VmOperations | test_18_DeployVM_NoFreeIPs_2_domain | 310.563
TestNetworksInAdvancedSG_VmOperations | test_20_DeployVM_SecGrp_sharedNetwork_1_default | 308.611
TestNetworksInAdvancedSG_VmOperations | test_20_DeployVM_SecGrp_sharedNetwork_2_custom | 305.574
TestNetworksInAdvancedSG_VmOperations | test_24_DeployVM_Multiple_Shared_Networks | 353.151
TestNetworksInAdvancedSG_VmOperations | test_25_Deploy_Multiple_VM_Different_Shared_Networks_Same_SG | 396.687
TestNetworksInAdvancedSG_VmOperations | test_26_Destroy_Deploy_VM_NoFreeIPs | 474.784
TestNetworksInAdvancedSG_VmOperations | test_31_Deploy_VM_multiple_shared_networks_sg | 355.199
TestNetworksInAdvancedSG_VmOperations | test_33_VM_Migrate_SharedNwSG_1_account | 0.000
TestNetworksInAdvancedSG_VmOperations | test**16_AccountSpecificNwAccess | 408.727
TestNetworksInAdvancedSG_VmOperations | test**17_DomainSpecificNwAccess | 400.732
TestSecurityGroups_BasicSanity | test_32_delete_default_security_group | 214.162
TestSharedNetworkOperations | test_34_restart_shared_network_sg_1_account | 261.479
TestSharedNetworkOperations | test_34_restart_shared_network_sg_2_domain | 258.813
TestSharedNetworkOperations | test_34_restart_shared_network_sg_3_zone | 258.858

Reply:
ok, let's keep as component tests
it is good you have run the tests and all passed @vishesh92